src/components/homepage/Slider.js

```diff
@@ -27,11 +27,14 @@ class Slider extends Component {
     getFrontPageImage(function (data) {
       if (data.length > 0) {
         var url = data[0].url;
+        console.log(url)
         if (url.indexOf('http') >= 0) {
           that.setState({ image: data[0].image, loading: false, url: url });
+
         }
-        else {
-          that.setState({ image: data[0].image, loading: false, url: null });
+        else{
+          that.setState({ image: slider1, loading: false, url: null }); // Currently this works but may need to think of a better way of doing it
```

Comment:
If we get a valid response from the API we should definitely use that image instead of the local one. As you suggest we should definitely clean this up a bit.

 I've also added the new image to Drupal.

I'll fix this up and remove the console log.